replica/database.cc

```diff
@@ -393,6 +395,24 @@ database::database(const db::config& cfg, database_config dbcfg, service::migrat
     if (_dbcfg.sstables_format) {
         set_format(*_dbcfg.sstables_format);
     }
+
+    _tp_timer.arm_periodic(std::chrono::seconds(_cfg.persistent_toppartitions_publish_interval_sec()));
+}
+
+void database::on_tp_timer() {
```

Comment:
Please avoid abbreviations like this in a general class like database.

Reply:
Fixed.